integration/README.md

```diff
@@ -0,0 +1,54 @@
+# Integration Tests
+
+This tests core use cases for u-root such as:
+
+- retrieving and kexec'ing a Linux kernel,
+- uinit (user init), and
+- running unit tests requiring root privileges.
+
+## Usage
+
+Run the tests with:
+
+    go test
+
+When the QEMU arch is not amd64, set the `UROOT_TESTARCH` variable. For
+example:
+    UROOT_TESTARCH=arm go test
+
+Currently, only amd64 and arm are supported.
+
+## Steps to get the kernel (bzImage)
+#Step 1: clone the open source linux directory:
+git clone https://github.com/torvalds/linux.git
+
+#Step 2: enter the linux directory and see the directories and files within it
```

Comment:
you don't need to say 'enter the directory'

Reply:
You're right. I had more details involved in the directions. I think I was more specific than needed to be.